tests/helpers/param.nim

```diff
@@ -0,0 +1,4 @@
+const fast {.strdefine.} = ""
```

Comment:
You can use booldefine, and that way in the .nimble, instead of having `-d:fast=true `, you could just put `-d:fast` and wouldnt need to do this comparison against `"true"`

Reply:
nice :)